Makefile

```diff
@@ -347,8 +355,13 @@ install:all $(PKGCONFBASE)
 	install -m644 libdeflate.h $(DESTDIR)$(INCDIR)
 	install -m755 gzip$(PROG_SUFFIX) \
 		$(DESTDIR)$(BINDIR)/libdeflate-gzip$(PROG_SUFFIX)
+ifneq ($(findstring -haiku,$(TARGET_MACHINE)),)
```

Comment:
Can you make this part conditional on `HARD_LINKS` being nonempty instead, similar to how `SHARED_LIB_SYMLINK` works just below?

Reply:
Had been thinking about that, but haven't figured that one out, thought about this as in the first part this is covered with HARD_LINKS but you didn't add some check here, could have a look for it (will get on that)

Reply:
`HARD_LINKS` was only unset before for Windows builds, where the `install` target isn't used.  If you want the `install` target to work for this platform, making it check `HARD_LINKS` is the right thing to do.

Reply:
My idea too, this was a first "quick" solution to our problem :) busy with some other projects but if time permits I get back on it soon :)